app/src/main/java/com/hammer/talkbbokki/data/repository/CommentRepositoryImpl.kt

```diff
@@ -0,0 +1,29 @@
+package com.hammer.talkbbokki.data.repository
+
+import com.hammer.talkbbokki.data.remote.TalkbbokkiService
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between